examples/host_func_test.go

```diff
@@ -47,34 +50,34 @@ func Test_hostFunc(t *testing.T) {
 		n, err := rand.Read(b)
 		require.NoError(t, err)
 		require.Equal(t, bufferSize, n)
+
+		expectedBase64String = base64.StdEncoding.EncodeToString(b)
 	}
 
 	r := wazero.NewRuntime()
 
-	env := &wazero.HostModuleConfig{Name: "env", Functions: map[string]interface{}{"get_random_string": getRandomString}}
+	env := &wazero.HostModuleConfig{Name: "env", Functions: map[string]interface{}{"get_random_bytes": getRandomBytes}}
 	_, err := r.NewHostModule(env)
 	require.NoError(t, err)
 
 	// Note: host_func.go doesn't directly use WASI, but TinyGo needs to be initialized as a WASI Command.
-	_, err = r.NewHostModule(wazero.WASISnapshotPreview1())
+	stdout := bytes.NewBuffer(nil)
+	_, err = r.NewHostModule(wazero.WASISnapshotPreview1WithConfig(&wazero.WASIConfig{Stdout: stdout}))
 	require.NoError(t, err)
 
 	module, err := wazero.StartWASICommandFromSource(r, hostFuncWasm)
 	require.NoError(t, err)
 
-	allocateBufferFn := module.Function("allocate_buffer")
-
-	// Implement the function pointer. This mainly shows how you can decouple a module function dependency.
```

Comment:
yeah this was the point, not just decoupling the dependency from a chicken-egg, but also from its signature requiring wazero types. Maybe I should have clarified that..